src/blocks/blocks/countdown/style.scss

```diff
@@ -2,30 +2,38 @@
 	--background-color: transparent;
 	--border-color: #000;
 	--border-radius: 0px;
+
 	--width: 100%;
 	--container-width: clamp(400px, var(--width) * 4, 100%);
 	--container-width-tablet: var(--container-width);
 	--container-width-mobile: var(--container-width-tablet);
+
 	--height: 100px;
-	--height-tablet: 100px;
-	--height-tablet: var(--height-tablet);
+	--height-tablet: var(--height);
 	--height-mobile: var(--height-mobile);
+
 	--border-style: solid;
 	--border-width: 2px;
 	--border-width-mobile: var(--height-tablet);
```

Comment:
And this not `--border-width-mobile`? Just double checking to make sure this was intended.

Reply:
@Soare-Robert-Daniel I don't know what that emoji means. Is it a "Yes, you're right" or "Nope, it's supposed to be that way." 😆

Reply:
-> "Yes, you're right"

Reply:
Ahaa, thanks for the quick fix.